cluster/common.sh

```diff
@@ -24,7 +24,7 @@ export CPU_PLAYER_CONTAINER=${CPU_PLAYER_CONTAINER:-"minigo-player"}
 # So, we prefix the project name to avoid collisions
 #
 # For more details, see https://cloud.google.com/storage/docs/best-practices
-export BUCKET_NAME=${BUCKET_NAME:-"${PROJECT}-minigo-v2-${BOARD_SIZE}"}
+export BUCKET_NAME=${BUCKET_NAME:-"${PROJECT}-minigo-v3-${BOARD_SIZE}"}
```

Comment:
maybe [v3] should also be an env variable?

Seems to me we should use more robust versions for our minigo versions. at least vMajor-Minor

Reply:
there's probably a bit of cleanup here.  I don't think everything in common.sh needs to have a default, and sometimes the 'see if it's set and don't change it if it is' isn't quite the right behavior when you want to source common.sh and have everything get updated.  Let's chat about it sometime.